script.js

```diff
@@ -0,0 +1,43 @@
+const button = document.querySelector('button');
+const title = document.querySelector('.title');
+const author = document.querySelector('.author');
+const content = document.querySelector('.books');
+
+const userData = JSON.parse(localStorage.getItem('formdata')).reverse();
+let count = userData.length;
```

Comment:
- [ ] Great work on setting the `userData` to the value you retrieve from the `localStorage`, however as you may notice when there's nothing yet in the localStorage we will get this error at the running time as the value is `null`. Kindly consider giving the `userData` a fallback value if the value from `localStorage` is `null`.
![image](https://github.com/ctokoli/Awesome_books/assets/97100091/f149916a-aad3-449f-8b51-aebf76ed4ed2)

Hint: You can achieve this by saying:
```suggestion
const userData = JSON.parse(localStorage.getItem('formdata')).reverse() || [];
```